app/controllers/wishlist_items_controller.rb

```diff
@@ -3,7 +3,7 @@ class WishlistItemsController < ApplicationController
 
   def index
     skip_authorization
-    @wishlist_items = WishlistItem.includes(:item, :wishlist).priority_order
+    @wishlist_items = WishlistItem.includes(:item, :wishlist).page(params[:page]).per(10)
```

Comment:
Will this work if we specify `default_per_page = 10` in the config file and remove `.per(10)`?

Reply:
Removed the `.per(10)` and using the config.yml instead for this